content/docs/learning-guide/tutorials/assets/mesh-assets.md

```diff
@@ -0,0 +1,67 @@
+
+    For this tutorial, you can use the default mesh group with all meshes selected.
+
+1. The customizations you make in Scene Settings are stored in a *sidecar file* with a `.assetinfo` extension. When Asset Processor detects a `.assetinfo` file, it uses the settings in the file to process the related source asset. This sidecar file is treated as a source dependency for the asset. This means that if the `.assetinfo` file is changed, the source asset will be reprocessed even if the source asset has not changed.
+
+    Let's add a modifier to customize how the asset is processed. Choose the **Add Modifier** button to view the mesh modifier list and select **Coordinate system change**
+
+    ![ Scene Settings meshes tab, adding mesh a modifier. ](/images/learning-guide/tutorials/assets/meshes-coordinate-system-change.png)
+
+1. The Coordinate system change mesh modifier is used to scale or transform the asset for those scenarios where the asset might be too small, too large, or incorrectly oriented in O3DE. By default, the modifier provides a single option to rotate the mesh 180 degrees. Activate the **Use advanced settings** toggle to expose the advanced modifier settings.
+
+    ![ Scene Settings meshes Coordinate system change modifier, Use advanced settings. ](/images/learning-guide/tutorials/assets/meshes-use-advanced-settings.png)
+
+1. Let's customize the scale of the asset. Set the **Scale** property to `5.0` to scale the asset to five times its size.
+
+{{< note >}}
+You can learn about the advanced settings properties of the Coordinate system change modifier, as well as the other available modifiers and their properties, in the [Scene Settings](/docs/user-guide/assets/scene-settings) topic of the **User Guide**.
+{{< /note >}}
+
+7. Choose the **Update** button at the bottom-right of Scene Settings. This creates or updates the `.assetinfo` sidecar file and triggers Asset Processor to reprocess the asset.
+
+8. **Left-click + drag** the source asset from Asset Browser into the viewport.
+
+    {{< image-width "/images/learning-guide/tutorials/assets/meshes-finished.png" "800" "Drag the mesh asset into the viewport">}}
+
+    When you drag the asset into the viewport, O3DE automatically creates an entity with a **Mesh** component that references the mesh product asset. If the source asset contains materials that have been processed, the materials are applied to the mesh by default.
+
+There are many options for customizing mesh processing, but there are limitations to the data that can be processed for meshes. Note that there are several product assets generated for the mesh listed below the source asset in Asset Browser. Each of these product assets contains some stream of data for the mesh. To learn more about the data that can be processed for a mesh and the limitations on the data that can be processed, refer to [Scene Format Support](/docs/user-guide/assets/scene-settings/scene-format-support) in the User Guide.

```

Comment:
Adding "that are" clarifies that it's the product assets that are listed below, not the mesh that is listed below. 
```suggestion
There are many options for customizing mesh processing, but there are limitations to the data that can be processed for meshes. Note that there are several product assets generated for the mesh that are listed below the source asset in Asset Browser. Each of these product assets contains some stream of data for the mesh. To learn more about the data that can be processed for a mesh and the limitations on the data that can be processed, refer to [Scene Format Support](/docs/user-guide/assets/scene-settings/scene-format-support) in the User Guide.

```